cmd/openshift-install/create.go

```diff
@@ -24,6 +24,8 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	clientwatch "k8s.io/client-go/tools/watch"
 
+	"github.com/openshift-metalkube/openshift-installer/pkg/asset/installconfig"
```

Comment:
I imagine these imports all need to be fixed to make CI work against this PR?

Reply:
Updated, they were fixed everywhere but here